functions/Get-DbaTcpPort.ps1

```diff
@@ -87,7 +87,7 @@ function Get-DbaTcpPort {
 							$tcp = $instance.ServerProtocols | Where-Object Name -eq Tcp
 							$ips = $tcp.IPAddresses
 
-							Write-Message -Level Verbose -Message "Parsing information for $($ips.count) IP addresses"
+							Write-Verbose "Parsing information for $($ips.count) IP addresses"
```

Comment:
It works!
I just suggest to add a comment before this line saying "do not change to Write-Message because bla bla bla" otherwise I think it will strike back 😛